Makefile

```diff
@@ -70,4 +70,4 @@ test-deps:
 	@which goveralls > /dev/null || \
 		(go get github.com/mattn/goveralls)
 
-travis-ci: lint build coveralls
+travis-ci: lint build coveralls release
```

Comment:
This is not CI anymore, lets rename release -> package to make it more understandable.

Reply:
Changed.